app/api/routes.py

```diff
@@ -17,25 +19,28 @@ def languages():
 
 
 def get_resources():
-    resources = {}
     try:
-        resources = Resource.query.all()
+        page = request.args.get('page', 1, type=int)
+        resource_paginator = Resource.query.paginate(page, Config.RESOURCES_PER_PAGE, False)
```

Comment:
Can you add an optional query parameter to override this `RESOURCES_PER_PAGE` not to exceed a maximum value?

Reply:
Ooh, good call. Should we call it `max_num`? So it will be like `api/v1/resources?page=2&max_num=10`

Reply:
Slack uses 
`limit`
twilio uses 
`PageSize`
I actually like `page_size` if you can find one that's more standard or better we go with that one. Don't want to bikeshed naming too much.